### ISSUE TITLE
mac vim *.py error

### ISSUE THREAD

**Issue by wqingzex** [spf13/spf13-vim#1026]
File "\<string\>", line 1, in <module>
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/wqz/.vim/bundle/python-mode/pymode/lint.py", line 9, in <module>
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pylama.lint.extensions import LINTERS
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/wqz/.vim/bundle/python-mode/pymode/libs/pylama/lint/extensions.py", line 30, in <module>
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    from pkg_resources import iter_entry_points
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/Users/wqz/.vim/bundle/python-mode/pymode/libs/pkg_resources/__init__.py", line 21, in <module>
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import io
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/io.py", line 51, in <module>
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
    import _io
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
ImportError: dlopen(/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so, 
2): Symbol not found: __PyCodecInfo_GetIncrementalDecoder
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  Referenced from: /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
  Expected in: flat namespace
Error detected while processing /Users/wqz/.vim/bundle/python-mode/autoload/pymode/lint.vim:
line    1:
 in /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so

# os
mac serria 10.12.3
python 2.7.13

**Comment by qiuwei** [comment]
Stop using spf13-vim, it's not maintained any more.

You can try spacemacs, which provides much more stable and powerful functionalities.

**Comment by Page-David** [comment]
But spacemacs is for emacs, isn't it?

**Comment by qiuwei** [comment]
@Page-David yes, it is. However, it has a vim emulation layer. For me, it feels really like using vim.

You might also want to try spacevim which is built upon vim, folllowing similar ideas from spacemacs.

**Comment by halfcrazy** [comment]
try manually install python-mode
git clone --recursive https://github.com/python-mode/python-mode.git
